packages/data-explorer/src/routes.ts

```diff
@@ -1,26 +1,36 @@
 import MainView from './views/MainView.vue'
+import DataRowEdit from '@/components/DataRowEdit.vue'
 
-const routes = [
+export const defaultRouteQuery:any = { page: 1, size: 20, hideSidebar: 'false', view: 'CardView' }
+
+export const routes = [
   {
-    path: '/:entity/:view',
+    path: '/explorer/:entity',
     component: MainView,
-    name: 'main-view'
+    name: 'de-view'
```

Comment:
why the 'de'; prefix , the whole app is in de data-explorer namespace , main-view matches the view name and seems more descriptive

Reply:
See comment above.